### PR TITLE
adding ESC to our main cloud getting started guides

### PR DESCRIPTION
**Pull request by interurban**
Adding more information about using ESC to our AWS, GCP, Azure and K8 getting started guides.

### CODE REVIEW

File: content/docs/clouds/aws/get-started/begin.md [hunk 1]
```diff
@@ -110,6 +110,10 @@ $ export AWS_PROFILE="<YOUR_PROFILE_NAME>"
 
 For additional information on setting and using AWS credentials, see [AWS Setup](/registry/packages/aws/installation-configuration/).
 
+{{% notes type="info" %}}
+Manually exporting AWS access keys can be tedious and risky, with frequent plaintext copy-pasting. With Pulumi ESC, you can dynamically retrieve short-lived AWS credentials via OpenID Connect, removing the need for manual exports and enhancing security. Learn more about [Pulumi ESC](/docs/esc/).
```
Comment (thtmnisamnstr) [inline content/docs/clouds/aws/get-started/begin.md:114]: Change "tedious and risky" to "prone to error and a security risk".

File: content/docs/clouds/aws/get-started/next-steps.md [hunk 2]
```diff
@@ -17,6 +17,20 @@ aliases:
 
 Congrats! You've deployed your first project on AWS with Pulumi. Here are some next steps, depending on your learning style.
 
+## Try Pulumi ESC (Environments, Secrets, and Configuration)
+
+In this guide, you used [stack configuration](/docs/concepts/config/) and environment variables to configure a single project and stack. Did you know you can also configure multiple Pulumi projects, and even other applications, with [Pulumi ESC](/docs/esc/)?
+
+Pulumi ESC introduces the concept of _environments_ --- managed collections of static and dynamic settings that you can use to configure any project, stack, application, or service, including with short-lived cloud credentials through OpenID Connect.
```
Comment (thtmnisamnstr) [inline content/docs/clouds/aws/get-started/next-steps.md:24]: I feel like you should start this paragraph off by describing Pulumi ESC. Something like, "Pulumi ESC is a cetralized secrets management and orchestration service. It introduces the concept..."

File: content/docs/clouds/aws/get-started/begin.md (hunk 1 above)
Comment (interurban) [reply content/docs/clouds/aws/get-started/begin.md:114]: Thx, added

File: content/docs/clouds/aws/get-started/next-steps.md (hunk 2 above)
Comment (interurban) [reply content/docs/clouds/aws/get-started/next-steps.md:24]: Adjusted intro

**Review by sean1588 — APPROVED** [review]

**Pull request merged**